src/cli/README_NETDATA.md

```diff
@@ -170,6 +170,39 @@ unpublish
 Done
 ```
 
+### full
```

Comment:
Add to command list? https://github.com/openthread/openthread/blob/main/src/cli/README_NETDATA.md#command-list